src/views/Profile/index.jsx

```
@@ -78,6 +109,34 @@ const Profile = () => {
 
+  const [deleteVideoModal, setDeleteVideoModal] = useState(false);
+  const [deletePicTourModal, setDeletePicTourModal] = useState(false);
+  const [deleteNewsModal, setDeleteNewsModal] = useState(false);
+  const [deleteLetterModal, setDeleteLetterModal] = useState(false);
+  const [deleteQafiModal, setDeleteQafiModal] = useState(false);
+  const [deleteEbicModal, setDeleteEbicModal] = useState(false);
+  const [deleteSportsModal, setDeleteSportsModal] = useState(false);
+  const [deleteCinematicModal, setDeleteCinematicModal] = useState(false);
+  const [deleteFanStarModal, setDeleteFanStarModal] = useState(false);
+  const [deleteKidVidsModal, setDeleteKidVidsModal] = useState(false);
+  const [deleteTvProgMaxModal, setDeleteTvProgMaxModal] = useState(false);
```

ESLint

[error] 'deleteTvProgMaxModal' is assigned a value but never used.

(no-unused-vars)